synapse/lib/stormtypes.py

```diff
@@ -1205,6 +1242,9 @@ def __init__(self, node, path=None):
             # TODO implement set()
         })
 
+    async def _derefGet(self, name):
```

Comment:
unused?

Reply:
Its part of a slightly more modular deref design and allows $node.props.foo